plugins/microsoft-calendar/README.md

```diff
@@ -0,0 +1,19 @@
+# microsoft-calendar
```

Comment:
We could probably improve this a lot so that people won't run into issues using this. It would be great to have step by step instructions as to how to install - the `yarn add` command for the package, where does this get added, etc.  

Also, looking over the code it assumes Azure AD auth has been setup, it would be good to include information about that here even if it just a link to the main documentation on this topic. Without this information the calendar won't work and people will log issues about it or simply not use the plugin, which would be a shame as it's pretty awesome!

Reply:
Here's an example you can use to help for the installation instructions: https://github.com/backstage/backstage/tree/master/plugins/azure-devops

And here's the docs for the Azure auth: https://backstage.io/docs/auth/microsoft/provider

Reply:
ok will update readme accordingly and push

Reply:
requested changes done @awanlin  @benjdlambert  are we good to integrate this?